### ISSUE TITLE
Cloudflared Local break existing cloudflare DNS

### ISSUE THREAD

**Issue by NasKar2** [brenner-tobias/addon-cloudflared#552]
### The problem

I installed cloudflared locally (recommended) and all worked but it broke my existing Cloudflare DNS setup.  My pfsense router would not update my WAN address in DDNS and my nextcloud couldn't be accessed outside my home.  After deleting the app and all entries in Cloudflare the DDNS and nextcloud worked again.  I installed Cloudflared with the remote option and everything works.  Must be an issue with the tunnels with local setup.  If it's not a bug I would mention it in the wiki.

### What version of Cloudflared has the issue?

4.2.12

### What type of installation are you running?

Home Assistant OS

### Steps to reproduce the issue

See the problem section

**Comment by brenner-tobias** [comment]
This sounds pretty strange. Was there an overlap of the domains you used for the external_hostname and your other domains? The tunnel does only create one CNAME for the external_hostname entry, thats it, no clue how this might interfere with your other domains so feels like a configuration issue

**Comment by NasKar2** [comment]
I'm a beginner when it comes to cloudflare.
I only have one domain registered with Namecheap with the DNS pointing to
cloudflare.  In my DNS entries I have one CNAME called cloud.mydomain.com
that I use for my nextcloud server that has ports open in my pfsense
router.  The router also has a DDNS service to keep cloudflare up to date
with my external IP address. When I did the local setup I saw a new entry
in the DNS section of cloudflare that used the tunnel and my DDNS no longer
worked and I couldn't access my nextcloud outside my home.  I deleted the
Cloudflared addon and Cloudflare tunnel and new tunnel DNS entry and my
DDNS started to work again and I could access nextcloud externally.  When I
used the remote setup everything worked.  My home assistant CNAME is
ha.mydomain.com.
